src/data/faq.json

```diff
@@ -2171,66 +2171,304 @@
                     "description": "See <a href='#glossary_signature'>Signature</a>"
+                },
+                {
+                    "term": "Digital Vaccination Certificate",
+                    "anchor": "digital_vaccination_certificate",
+                    "description": "See <a href='#glossary_digital_certificate'>Digital Certificate</a>"
                 }
             ],
             "E": [
+                {
+                    "term": "Effective validity",
+                    "anchor": "effective_validity",
+                    "description": "We refer to the duration of vaccination protection as 'effective validity'. The effective validity of the vaccination thus counts from the last vaccination date. While in Germany the professional validity is currently set at one year, the validity of the vaccination protection in other countries may differ from this. In the case of a <a href='#glossary_recovery_certificate'>recovery certificate</a>, the effective validity is set at 'date of the person's first positive <a href='#glossary_pcr_test'>PCR test</a> result' plus 180 days, according to the EU eHealth Network."
+                },
+                {
+                    "term": "Error report",
+                    "anchor": "error_report",
+                    "description": "A detailed report of a malfunction that has occurred, which is sent to the responsible developers so that they can fix the described error."
+                },
                 {
                     "term": "EU DCC",
                     "anchor": "eu_dcc",
                     "description": "See <a href='#glossary_dcc'>DCC</a>"
+                },
+                {
+                    "term": "European Federation Gateway Service (EFGS)",
+                    "anchor": "efgs",
+                    "description": "A central infrastructure that is used to exchange and check <a href='#glossary_diagnosis_key'>diagnosis keys</a> across national borders."
+                },
+                {
+                    "term": "Event registration",
+                    "anchor": "event_registration",
+                    "description": "Register for an event or a visit to a location."
+                },
+                {
+                    "term": "Exposure Detection",
+                    "anchor": "exposure_detection",
+                    "description": "List of short-lived random Bluetooth IDs received and temporarily stored in the operating system memory. The encounter record is read as part of the risk determination during the risk check. All short-lived random Bluetooth IDs older than 14 days are automatically deleted. The encounter record can also be deleted in its entirety at any time by individuals using the app. When the CWA is uninstalled, the encounter record is automatically deleted."
+                },
+                {
+                    "term": "Exposure Notification Express (ENE)",

```

Comment:
ENE was developed and made available by Google and Apple. See https://developers.google.com/android/exposure-notifications/en-express

Since only iOS is mentioned here it sounds like it is only available from Apple, which is not true, it is also available from Google.

Reply:
Solved by just refering to the FAQ article, thanks!